### PR TITLE
Get contract addresses from deployment info by default

### LINKED ISSUE #153: Get contract addresses from contract package
## Problem Definition

Currently the MS and RC need the contracts to be defined in the CLI. The contract addresses should be (and are) defined in the raiden-contracts package. We should take them from there by default but let them be overridden.

- [ ] Get contract addresses for service onctract from `raiden-contracts`  package by default.
- [ ] Let them be overridden by user

### PR DESCRIPTION
**Pull request by palango**
Fixes #153 

The PFS cli tests still need some work, but have a look if you like the general approach.

### CODE REVIEW

File: src/raiden_libs/contract_info.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                user_deposit_contract_info['block_number'],
+            ) - start_block_safety_margin,
+        )
+        return {
+            CONTRACT_TOKEN_NETWORK_REGISTRY: registry_address,
+            CONTRACT_MONITORING_SERVICE: monitor_address,
+            CONTRACT_USER_DEPOSIT: udc_address,
+            START_BLOCK_ID: given_start_block if all_addresses_given else contracts_start_block,
+        }
+    except ValueError:
```
Comment (karlb) [inline src/raiden_libs/contract_info.py]: That's a pretty large try block for such a generic exception as `ValueError`. We might quickly get a "No deployed contracts were found at the default registry" message when introducing a bug that causes an unintended `ValueError`.

File: src/raiden_libs/contract_info.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        monitor_address = monitor_contract_address or monitor_contract_info['address']
+        udc_address = user_deposit_contract_address or user_deposit_contract_info['address']
+
+        contracts_start_block = max(
+            0,
+            min(
+                token_network_registry_info['block_number'],
+                monitor_contract_info['block_number'],
+                user_deposit_contract_info['block_number'],
+            ) - start_block_safety_margin,
```
[the remark this answers is not in the document]
Comment (palango) [reply src/raiden_libs/contract_info.py]: I remember there were cases where ethereum nodes needed some margin in order to correctly return events, but i'm not sure. I'll test it.

File: src/raiden_libs/contract_info.py (hunk 1 above)
Comment (palango) [reply src/raiden_libs/contract_info.py]: I simplified the function, please have another look.

File: src/raiden_libs/contract_info.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (palango) [reply src/raiden_libs/contract_info.py]: Seems like this stuff works now, removed the margin argument.

**Pull request merged**